permission_validators/main/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "iroha_main_validator"
+version = "2.0.0-pre-rc.13"
+authors = ["Iroha 2 team <https://github.com/orgs/soramitsu/teams/iroha2>"]
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+# A smartcontract should be linked dynamically so that it may link to functions exported
+# from the host environment. Also, host environment executes the smartcontract by
+# calling the function which smartcontract exports (entry point of execution)
+crate-type = ['cdylib']
+
+[dependencies]
+iroha_wasm = { path = "../../wasm", features = ["debug"]}
```

Comment:
you have activated `debug` in every permission validator?

Reply:
Yeah. Probably oneday it should be replaced with logging system you are working on

Reply:
hm, let me put it this way. This feature should not be active in `release` mode?

Reply:
All this feature does is printing message on iroha side when something panics inside wasm-blob. I'm not sure this is bad for `realease` mode, cause program in `release` can fail too

Reply:
We should have a working logging system for smartcontracts. One of the chief critiques of Ethereum is that their triggers and executables are nearly impossible to debug, thanks to virtually no feedback. Until we know exactly that these permission validators behave correctly, I'd leave the logging on.